lib/app/custom_widgets/match_league_and_timer_badge.dart

```diff
@@ -56,33 +55,43 @@ class MatchLeagueInfoAndTimerWidget extends StatelessWidget {
             )
           ],
         ),
-        Container(
-          height: 30,
-          width: 50,
-          decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(24),
-            color: AppColors.liveTimerBackgroundGreen,
-          ),
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceAround,
-            children: [
-              CircleAvatar(
-                radius: 5,
-                backgroundColor: matchStatusShort == 'TBD' || matchStatusShort == 'NS' || matchStatusShort == 'SUSP'
-                    ? Colors.red
-                    : Colors.green,
+        matchStatusShort != 'FT'
```

Comment:
Consider moving 'FT' to separate class with some kind of values and give it more descriptive name because hardcoded 'FT' does not tell much when you look at the code.

Reply:
I will do that in a short while, same with placeholders and default graphics of whole app